vitest.shared.ts

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+    test: {
+        globals: true,
+        include: ['**/*.{test,spec}.{js,ts}'],
+        snapshotFormat: {
+            printBasicPrototype: true,
+            callToJSON: true,
+        },
+        coverage: {
+            exclude: [
+                '/node_modules/',
+                '/fixtures/',
+                '/dist/',
+                // Ignore helper files like test-utils.ts that might exist alongside spec files
+                '/__tests__/',
+            ],
+            thresholds: {
+                branches: 80,
+                functions: 90,
+                lines: 90,
+            },
+            reporter: ['clover', 'json', 'lcov', 'text', ['text', { file: 'coverage.txt' }]],
```

Comment:
> Looks like Vitest doesn't support the `txt` coverage output, so our custom GitHub step summary coverage report is failing. We may have to just get rid of it, since the only other option seems to be a custom GitHub Action (which feels too risky for the benefit).

@nolanlawson https://github.com/salesforce/lwc/pull/4396#issuecomment-2251422693

Reply:
Turns out this config was getting ignored. 🫣 Fixed